src/components/MobileMenu/mobileMenuStyling.css

```diff
@@ -1,56 +1,54 @@
-import styled from 'styled-components';
-import {Link as LinkS} from 'react-scroll';
-import {FaTimes} from 'react-icons/fa';
 
-export const MobileMenuContainer = styled.aside`
-    background: green;
-    display: block;
+.menu-container{
+    display: none;
     position: fixed;
     z-index: 999;
     width: 100%;
     height: 100%;
     background: #0d0d0d;
+    opacity: 0.9;
     align-items: center;
     top: 0;
     left: 0;
     transition: 0.3s ease-in-out;
-    opacity: ${({isOpen}) => (isOpen ? '100' : '0')};
-    top:${({isOpen}) => (isOpen ? '0' : '-100')};
-    @media screen and (max-width: 480px) {
+    /* opacity: ${({isOpen}) => (isOpen ? '100' : '0')};
+    top:${({isOpen}) => (isOpen ? '0' : '-100')}; */
```

Comment:
Remove if not needed

Reply:
.menu-container—open {
Opacity: 100
Top: -100
}

<aside className={`menu-container ${isOpen ? ‘menu-container—open’ : ‘’}`}>

Reply:
aside className={`menu-container ${isOpen ? ‘menu-container—open’ : ‘’}`}